translations/setup.tex/pl.po

```diff
@@ -125,6 +127,59 @@ msgid ""
 "\\end{enumerate}\n"
 "\n"
 msgstr ""
+"\\begin{enumerate}\n"
+"  \\item Wybierz Scenariusz z Księgi Misji.\n"
+"    Jako pierwszy polecamy Scenariusz \"Nowy Wspaniały Świat\" (zob. str. 7 w Księdze Misji).\n"
```

Comment:
```suggestion
"    Jako pierwszy polecamy Scenariusz \enquote{Nowy Wspaniały Świat} (zob. str. 7 w Księdze Misji).\n"
```
Taki cudzysłów jest niezgody z polską typografią i LaTeX ma problem ze spacją po zamknięciu. To powinno zadziałać.